medicines/docs/contributor_guidelines/code-of-conduct.md

```diff
@@ -0,0 +1,89 @@
+# Code of conduct for `MHRA`
```

Comment:
How come we have two of these? Would contributor guidelines not be global for the repository?

Reply:
Contributor guidelines will eventually feature style guides and other elements that may be quite different from project to project, so giving the ability for each project to set their own criteria.